examples/huggingface/gpt-neo.py

```diff
@@ -0,0 +1,26 @@
+from pipeline import Pipeline, Variable
+from pipeline.objects.huggingface.TransformersModelForCausalLM import (
+    TransformersModelForCausalLM,
+)
```

Comment:
Seeing as you have the stuff in `huggingface.__init__` this can use it.

```suggestion
from pipeline.objects.huggingface import TransformersModelForCausalLM
```